### PR TITLE
statistics: add some key info in sync load timeout log

### LINKED ISSUE #49631: Resolve most of timeout sync load stats request in mertrics
## Background

The metrics of sync_load_timeout is similar with sync_load which means that most of sync load stats request are timeout.
But it is too difficult to investigate this phenomenon.

The metrics :
image

Only the column id and column name in the timeout log

image

## Enhancement

Need more information about the sync load timeout request such as global table id, query id etc

### PR DESCRIPTION
**Pull request by elsa0520**
### What problem does this PR solve?

Issue Number: close #49631

Problem Summary:

### What changed and how does it work?

Add table id in timeout sync load log

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by winoros — APPROVED** [review]

File: pkg/statistics/column.go [hunk 1]
```diff
@@ -168,7 +168,9 @@ func (c *Column) IsInvalid(
 		if (!c.IsStatsInitialized() || c.IsLoadNeeded()) && stmtctx != nil {
 			if stmtctx.StatsLoad.Timeout > 0 {
 				logutil.BgLogger().Warn("Hist for column should already be loaded as sync but not found.",
-					zap.String(strconv.FormatInt(c.Info.ID, 10), c.Info.Name.O))
+					zap.String("table_id", strconv.FormatInt(c.PhysicalID, 10)),
```
Comment (hawkingrei) [inline pkg/statistics/column.go:171]: You can replace it with ```zap.Int```.

Comment (elsa0520) [reply pkg/statistics/column.go]: Fixed

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**